cairo-vm-cli/src/main.rs

```diff
@@ -160,6 +199,11 @@ fn run(args: impl Iterator<Item = String>) -> Result<(), Error> {
         memory_writer.flush()?;
     }
 
+    #[cfg(feature = "with_tracer")]
+    if args.tracer.is_some() && args.tracer.unwrap() {
```

Comment:
You can simplify this with:
```suggestion
    if args.tracer.unwrap_or(false) {
```

Reply:
Fixed